Week2/Account/Account.java

```diff
@@ -0,0 +1,15 @@
+package Week2.Account;
+
+import java.math.BigDecimal;
+
+public interface Account { //
+    String getAccountInfo();
+    void withdraw(BigDecimal value);
+    void deposit(BigDecimal value);
+    String getAccountNumber();
+    AccountType getAccountType();
+    BigDecimal getBalance();
+    void eraseStringBuilder();
+    boolean isActivated();
+
+}
```

Comment:
```suggestion
}

```
EOL!
https://avocado12-note.tistory.com/11